benchmarks/time/src/main/scala/strawman/collection/immutable/ArrayBaselineBenchmark.scala

```diff
@@ -0,0 +1,143 @@
+package strawman.collection.immutable
+
+import java.util.concurrent.TimeUnit
+
+import org.openjdk.jmh.annotations._
+import org.openjdk.jmh.infra.Blackhole
+import strawman.collection.{ArrayView, View}
+import scala.{Any, AnyRef, Int, Long, Unit, Array}
+import scala.Predef.intWrapper
+
+@BenchmarkMode(scala.Array(Mode.AverageTime))
+@OutputTimeUnit(TimeUnit.NANOSECONDS)
+@Fork(1)
+@Warmup(iterations = 12)
+@Measurement(iterations = 12)
+@State(Scope.Benchmark)
+class ArrayBaselineBenchmark {
+
+  @Param(scala.Array("282", "73121", "7312102"))
+  var size: Int = _
+
+  @Param(scala.Array("39"))
+  var vLoSize: Int = _
+
+  var shortRangingFactor : Int = (size * 0.2).toInt
+
+  var v: Array[Long] = _
+  var vLo: Array[Long] = _
+
+  @Setup(Level.Trial)
+  def initData(): Unit = {
+
+    def fillArray(range: Int) = {
+      val array = new Array[Long](range)
+      var i = 0
+      while (i < range) {
+        array(i) = i.toLong
```

Comment:
If you're benchmarking primitives, it doesn't matter what the values are.  If you're benchmarking something that boxes, you probably should be using `String` instead of `Int` for your base case (and then when testing boxing you can decide whether it makes more sense to use values that may be cached, e.g. `2: Object` or not.